src/components/DocumentationTopic/DecoratedTopicTitle.vue

```diff
@@ -10,12 +10,16 @@
 
 <template>
   <code class="decorated-title">
-    <component
+    <template
       v-for="(token, i) in tokens"
-      :class="[classFor(token), emptyTokenClass(token)]"
-      :is="componentFor(token)"
-      :key="i"
-    >{{ token.text }}</component>
+    >
+       <component
+        :class="[classFor(token), emptyTokenClass(token)]"
+        :is="componentFor(token)"
+        :key="i"
+      >{{ token.text }}</component>
+      <wbr :key="`wbr-${i}`"/>
```

Comment:
I'm not totally understanding the scenario where the `<wbr>` elements from the individual token components aren't sufficient, but I'm not seeing any regressions introduced by adding the additional one here, so this change seems ok to me.